requirements.txt

```diff
@@ -0,0 +1,8 @@
+-e git+http://github.com/openaustralia/scraperwiki-python.git@morph_defaults#egg=scraperwiki
```

Comment:
Interesting. Listing this in *requirements.txt* actually breaks the scraper on ScraperWiki, maybe because their platform uses a different version.

Will the scraper work on morph.io without this? I know the master branch of this repo works without a *requirements.txt* at all, maybe because something special is enabled for this morph.io scraper since it was migrated from ScraperWiki using their one-click migration tool.